server/routes/activitiesMigration/activitiesMigrationController.ts

```diff
@@ -158,7 +155,7 @@ export default class ActivitiesMigrationController {
       }),
 
       this.nomisPrisonerService
-        .findActivitiesSuspendedAllocations(filter, await activityCategoriesPromise, context(res))
+        .findActivitiesSuspendedAllocations(filter, context(res))
```

Comment:
The main change is to stop sending the activity categories as excluded program codes to nomis prisoner api - these are no longer used and will be removed from the API.